### DISCUSSION TITLE
ILS doesn't seem to be working correctly

### DISCUSSION THREAD

**Discussion by Sky173** [flybywiresim/aircraft#3593] [Help]
I hope someone has run into this issue and can assist.

When I'm descending to my destination, I'll choose my approach. Today this occurred at two different locations. (The first was ILS31R @ KDFW via TULOY and the second was on approach @ KATL ILS9L).

I enter the (ILS) runway information into the MCDU but the magenta approach diamonds don't show up on the panel. I try to manually change the radio to the proper frequency, but it wont allow me to access the radio. I can't change the frequency at all. This has been on-going for over a week now. I haven't had an issue like this until then. Sometimes it works, Other times it doesn't. Am I missing something? Here's a video of my approach to Atlanta. I started recording as soon as I saw that this was an issue tonight.
https://youtu.be/0NCas-_qJoQ

In the video, I'm not flying at all until the last minute when I have to descend to land. Any lateral adjustments in flight are done by the aircraft. Not me... I've tried entering different ILS runways in the MCDU but that doesn't always work either.

It occurs with all three versions of the aircraft (Stable, Dev, and Exp)

Thanks for everything you guys are doing. I'm really enjoying this aircraft. Love what you're doing. Cheers.

**Comment by robinottawa** [comment]
Me too. The first time I was able to enter the correct ILS radio freq, but not the next time. 

I meant to look at the freq before I started down but haven't had the time yet.

**Comment by Sky173** [comment]
I don't know if this will help with the ILS issue in my original post, but here's a full flight video of what happened today. It occurred as I tried to copy the exact flight plan that caused the original issue at KDFW. I began recording as soon as I created the flight plan in the sim. I know it can be boring to watch the entire thing but hopefully it will show exactly what I did and how I managed to get around the problem (this time).

Here's the full video: https://youtu.be/oebl-Km2I-g?t=515

This video was shot using the dev version, but it does occur with the stable version as well. Cheers!
I left in the nose-dive on touchdown, but I know that's for another bug post I saw here. :)
